### PR TITLE
[ADD] l10n_ar: AR fields in portal

### PR DESCRIPTION
**Pull request by zaoral**
### Description of the issue/feature this PR addresses:

Add the Argentinean fields in the my/account porta

### Current behavior before PR:

Do not show AR fields

### Desired behavior after PR is merged:
Now we have AR fields
my-portal-preview

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by zaoral** [comment]
Hi @odoo/rd-security
We have a security risk in this PR
Please help us to know which should be the better approach to avoid the problem
https://runbot.odoo.com/runbot/build/50086533

Best Regards

**Comment by adgu-odoo** [comment]
Hello @fvz-odoo , did you test this part of the pos argentinian task (the user portal) ?

File: addons/l10n_ar/controllers/portal.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        response = super().account(redirect=redirect, **post)
+        response.qcontext.update({
+            'identification_types': request.env['l10n_latam.identification.type'].sudo().search([('l10n_ar_afip_code', '!=', False)]),
+            'responsibility_types': request.env['l10n_ar.afip.responsibility.type'].sudo().search([]),
+            'identification': post.get('l10n_latam_identification_type_id'),
+            'responsibility': post.get('l10n_ar_afip_responsibility_type_id')})
+        return response
+
+    def _vat_validation(self, data, error, error_message, partner_creation=False):
```
Comment (adgu-odoo) [inline addons/l10n_ar/controllers/portal.py:33]: This function does not exist so it is never called and you cannot override it

**Comment by vbe-odoo** [comment]
Moved to draft to keep working on it later (separately form the PoS PR.

**Comment by zaoral** [comment]
Hi @adgu-odoo 
Thank you for the review.

Not sure if it was completely tested but since I checked your message early today it seems that needs to be checked.
We discussed this with the LATAM Odoo team and we will leave this functionality on a Draft PR at the moment since actually is not really required to merge the POS module.

The POS module was moved to the community version PR https://github.com/odoo/odoo/pull/133881

Best Regards

**Comment by adgu-odoo** [comment]
> Hi @adgu-odoo Thank you for the review.
> 
> Not sure if it was completely tested but since I checked your message early today it seems that needs to be checked. We discussed this with the LATAM Odoo team and we will leave this functionality on a Draft PR at the moment since actually is not really required to merge the POS module.
> 
> The POS module was moved to the community version PR #133881
> 
> Best Regards

Hi @zaoral , that is ok for me. I just put another comment as I wanted to finish the review of this (I still had one file to look at when I left yesterday). We will have a talk with Fernando on Wednesday to discuss a bit of the POS PR and if everything is fine we will merge it on Wednesday. Have a good day

**Pull request closed without merging**